includes/options/class-amp-options-manager.php

```diff
@@ -31,6 +31,7 @@ class AMP_Options_Manager {
 		'disable_admin_bar'       => false,
 		'all_templates_supported' => true,
 		'supported_templates'     => array( 'is_singular' ),
+		'enable_response_caching' => true,
```

Comment:
The default value here can be the return value of `wp_using_ext_object_cache()`.

Reply:
One way to accomplish this is to set the default state within `get_options()`:

```
self::$defaults['enable_response_caching'] = wp_using_ext_object_cache();
return array_merge( self::$defaults, $options );
```